dev-environment/nginx/templates/default.conf.template

```diff
@@ -18,36 +18,41 @@ upstream bucket {
     server ${BUCKET_URL};
 }
 
+upstream pmml {
+    server ${PMML_SAMPLES_URL};
+}
```

Comment:
You can pass to `${PMML_SAMPLES_URL}` directly, without adding an upstream.

Reply:
but it will not be possible with nussknacker (which is defined in separate docker-compose and that is why I did all these changes - to make it possible for nussknacker machines to comunicate with pmml data server)

Reply:
So something like this won't work?:
```
server {
    listen ${PMML_SAMPLES_PORT};
    server_name pmml-samples;

    location / {
        proxy_pass ${PMML_SAMPLES_URL};
    }
}
```